README.md

```diff
@@ -111,6 +111,8 @@ VALSinglePromptSecureEnclaveValet *mySecureEnclaveValet = [[VALSinglePromptSecur
 
 This instance also stores and retrieves data in the Secure Enclave, but does not require the user to confirm their presence each time data is retrieved. Instead, the user will be prompted to confirm their presence only on the first data retrieval. A `VALSinglePromptSecureEnclaveValet` instance can be forced to prompt the user on the next data retrieval by calling the instance method `requirePromptOnNextAccess`.
 
+**In order for your app to officially support Face ID, you must set a value for the Privacy - Face ID Usage Description (NSFaceIDUsageDescription) key in your app's Info.plist.**
```

Comment:
I'd say: "In order for your customers not to receive a prompt that your app does not yet support Face ID, you must ..."

Also, curly quotes 😉